### PR TITLE
docs: update cloud data sources README links

### PR DESCRIPTION
**Pull request by vickyyyyyyy**
**What this PR does / why we need it**:

The documentation for the plugins are outdated.

- The Cloud Monitoring link redirected to `7.5` of the documentation: https://grafana.com/docs/grafana/latest/features/datasources/cloudmonitoring/
- The CloudWatch link gave a 404 error
- There was no README for Azure Monitor

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

TIA 🙏

### CODE REVIEW

**Review by connorlindsey — APPROVED** [review]

**Review by andresmgot — APPROVED** [review]

File: public/app/plugins/datasource/cloudwatch/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# CloudWatch Data Source -  Native Plugin
+# CloudWatch Data Source - Native Plugin
```
Comment (achatterjee-grafana) [inline public/app/plugins/datasource/cloudwatch/README.md:1]:
```suggestion
# CloudWatch data source - native plugin
```

File: public/app/plugins/datasource/cloud-monitoring/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Google Cloud Monitoring Data Source - Native Plugin (formerly named Stackdriver)
 
-Grafana ships with built-in support for Google Cloud Monitoring. You just have to add it as a data source and you will be ready to build dashboards for your Cloud Monitoring metrics.
+Grafana ships with **built in** support for Google Cloud Monitoring. You just have to add it as a data source and you will be ready to build dashboards for your Cloud Monitoring metrics.
```
Comment (achatterjee-grafana) [inline public/app/plugins/datasource/cloud-monitoring/README.md:3]:
```suggestion
Grafana ships with **built in** support for Google Cloud Monitoring. Simply add it as a data source and you are ready to build dashboards for your Cloud Monitoring metrics.
```

File: public/app/plugins/datasource/cloudwatch/README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 
 Grafana ships with **built in** support for CloudWatch. You just have to add it as a data source and you will be ready to build dashboards for your CloudWatch metrics.
```
Comment (achatterjee-grafana) [inline public/app/plugins/datasource/cloudwatch/README.md:3]:
```suggestion
Grafana ships with **built in** support for CloudWatch. Simply add it as a data source and are ready to build dashboards for your CloudWatch metrics.
```

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Azure Monitor Data Source - Native Plugin
```
Comment (achatterjee-grafana) [inline public/app/plugins/datasource/grafana-azure-monitor-datasource/README.md:1]: Requesting same changes as above.

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: public/app/plugins/datasource/grafana-azure-monitor-datasource/README.md (hunk 4 above)
Comment (daniellee) [reply public/app/plugins/datasource/grafana-azure-monitor-datasource/README.md:1]:
```suggestion
# Azure Monitor data source - native plugin
```

File: public/app/plugins/datasource/cloud-monitoring/README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Google Cloud Monitoring Data Source - Native Plugin (formerly named Stackdriver)
```
Comment (daniellee) [inline public/app/plugins/datasource/cloud-monitoring/README.md:1]:
```suggestion
# Google Cloud Monitoring data source - native plugin (formerly named Stackdriver)
```

**Review by daniellee — APPROVED** [review]

**Pull request merged**